### DISCUSSION TITLE
Diffusers live preview

### DISCUSSION THREAD

**Discussion by Arctomachine** [vladmandic/sdnext#2820] [Q&A]
I have it enabled, but it is not working with diffusers backend. It only shows already generated image (just before vae) and end result.
Is it possible to make it work?

**Comment by vladmandic** [comment]
all types of live preview work for most people. if you have a problem, document it full and create an issue.

**Reply by Arctomachine** [reply]
Where should I start looking into this? There are no errors or warnings about in terminal
